src/CommunityToolkit/Xamarin.CommunityToolkit/Views/SideMenuView/SideMenuView.shared.cs

```diff
@@ -87,6 +87,9 @@ public class SideMenuView : BaseTemplatedView<AbsoluteLayout>
 		public static readonly BindableProperty MenuGestureEnabledProperty
 			= BindableProperty.CreateAttached(nameof(GetMenuGestureEnabled), typeof(bool), typeof(SideMenuView), true);
 
+		public static readonly BindableProperty MainViewScaleFactorProperty
+			= BindableProperty.CreateAttached(nameof(GetMainViewScaleFactor), typeof(double), typeof(SideMenuView), 1.0);
```

Comment:
Didn't know that Forms remove the Get prefix 
![](https://media3.giphy.com/media/vSSdLSLbGIXio/giphy.gif)

Reply:
What do you mean?)

Reply:
@AndreiMisiukevich There you typed `nameof(GetMainViewScaleFactor)` and on xaml you use `SideMenuView.MainViewScaleFactor` there's no `Get` prefix

Reply:
@pictos I believe Xaml works with the name of BindableProperty. In this case, it's **MainViewScaleFactorProperty**
And it just removes postfix "Property".

Reply:
@AndreiMisiukevich makes sense...